build.sh

```diff
@@ -1,26 +1,29 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
Why did you remove -u here?

Reply:
Because it didn't let me handle the possible options _friendly_.
Or to but it different, most of the conditionals didn't work with -u since they just throw an error and exit the script if a parameter wasn't set. But I also changed that at some places.

Reply:
Hmm, I think i can reintroduce -u and keep the friendly behavior....
Wait 4 the next commit.